src/huggingface_hub/hf_api.py

```diff
@@ -10453,6 +10453,525 @@ def run_uv_job(
             token=token,
         )
 
+    def schedule_job(
```

Comment:
personal preference here, "create" is explicit and pairs naturally with list, inspect and delete
```suggestion
    def create_scheduled_job(
```